packages/web-client/app/components/card-pay/dashboard-panel/section/index.gts

```diff
@@ -0,0 +1,52 @@
+import Component from '@glimmer/component';
+import cn from '@cardstack/boxel/helpers/cn';
+import not from 'ember-truth-helpers/helpers/not';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+import { Section } from '../panel';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    section: Section;
+  }
+  Blocks: {
+    cta: []
+    disclaimer: []
+  }
+}
+
+export default class CardPayDashboardPanelSection extends Component<Signature> {
+  <template>
+    <section class={{cn "dashboard-panel-section" dashboard-panel-section--bottom-aligned=(not @section.title) dashboard-panel-section--has-disclaimer=(has-block "disclaimer")}} ...attributes>
+      <div class="dashboard-panel-section__body">
+        {{#if @section.icon}}
+          {{svgJar @section.icon role="presentation" width="72"}}
```

Comment:
Oh, I didn't realise that `alt` does not work for inline svg. We should probably reconsider the use of `role=presentation` too (in another PR), it may not the right way to get inline svgs ignored by assistive tech.

Reply:
I'm open to changes around this in a separate PR.